src/app/pages/offline-map/offline-map.page.html

```diff
@@ -23,22 +23,28 @@
     ></app-map>
   </div>
 </ion-content>
-<ng-container *ngIf="downloadAndUnzipProgress$ | async as items">
-  <ion-footer *ngIf="items.length > 0">
+<ion-footer>
+  <ion-list>
+    <ion-item *ngIf="packageTotals$ | async as packageTotals" (click)="toggleExpand()" lines="full">
+      <ion-label class="package-count">{{ 'OFFLINE_MAP.PACKAGE_COUNT' | translate:{packageCount: packageTotals.numPackages} }}</ion-label>
+      <ion-label>
+        <span class="packages-size">{{ packageTotals.spaceUsed }}</span>&nbsp; 
+        <span class="space-available">({{ 'OFFLINE_MAP.SPACE_AVAILABLE' | translate:{spaceAvailable: getSpaceAvailable()} }})</span>
+      </ion-label>
+      <ion-icon *ngIf="isExpandable$ | async" slot="end" name="chevron-up-circle" class="expand-icon"></ion-icon>
+      <ion-icon *ngIf="isCollapsible$ | async" slot="end" name="chevron-down-circle" class="expand-icon"></ion-icon>
+    </ion-item>
     <div class="footer">
-      <ion-list>
-        <ion-list-header>
-          <ion-label>{{ 'OFFLINE_MAP.DOWNLOAD_LIST_MAP_PACKAGE_HEADER' | translate }}</ion-label>
-        </ion-list-header>
-        <ion-item (click)="showPackageModalForPackage(item)" *ngFor="let item of items">
+      <ng-container *ngIf="expanded && downloadAndUnzipProgress$ | async as items">
+        <ion-item (click)="showPackageModalForPackage(item)" *ngFor="let item of items" lines="full">
           <ion-label>{{ item.name }}</ion-label>
           <ion-label>{{ humanReadableByteSize(item.size) }}</ion-label>
           <ion-label *ngIf="isDownloading(item)">
             ({{getPercentage(item) +'%' }})</ion-label>
           <ion-icon slot="end" *ngIf="item.error" name="warning-outline"></ion-icon>
```

Comment:
Lurer på om vi skulle prøvd med `[style.visibility]="item.error ? : 'visible' : 'hidden'"` her, evt `[hidden]="item.error"` hvis det går an, med hensikt å unngå at størrelse og prosent hopper eller flytter seg til venstre ved feil, som i skjermbildene i saken her.

Reply:
Er enig i at det ser litt teit ur. Fikk ikke noen av forslagene dine å virke, så da valgte jeg å skjule søplekassa hvis det feiler. Da får man se enten søplekasse eller fareskilt. Ulempen er at det blir et ekstra klikk inn på detaljsida for å kvitte seg med pakka som feilet, men tenker det kan være greit, så får man se feilmeldinga og kan prøve på nytt.

Reply:
Ser jeg hadde et kolon for mye der, så det var kanskje derfor? Uansett, høres greit ut dette.